tracks/rust/exercises/pythagorean-triplet/mentoring.md

```diff
@@ -0,0 +1,74 @@
+### Concepts
+
+- Option
+- Math
+
+### Reasonable solutions
+
+A reasonable solution should do the following:
+
+- Use `a.pow(2)` rather than `a*a`
+- not use return for the last line of the function
+- Not use floating point values, which introduce the posibility of rounding errors
+- Not introduce posibility of integer underflow.
+- Accounts for possibly not finding a triplet and returns None, not Some(0)
```

Comment:
Your example returns a `HashSet` and not an `Option` I'm not sure if this bullet point is valid in the context of the exercise.